gammapy/catalog/hess.py

```diff
@@ -516,6 +517,10 @@ def spatial_model(self):
             model = GaussianSpatialModel(
                 lon_0=glon, lat_0=glat, sigma=d["Size"], frame="galactic"
             )
+            if not np.isnan(d["GLON_Err"]):
```

Comment:
You have this `if np.isnan` code  many times.
Can this be avoided?

I think just passing `NaN` along is fine, just remove the ifs?